LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/ric-in-monospace.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/#font-size-prop">
+<link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
+<link rel="match" href="ric-in-monospace-ref.html">
+
+<p>The following two lines should look exactly the same.</p>
+
+<div style="font-family: monospace; font-size: 1.5em">
+  <div style="font-family: sans-serif; font-size: 1ric">
+    Text.
+  </div>
+</div>
+
+<div style="font-family: sans-serif; font-size: 1ric">
+  Text.
+</div>
```

Comment:
Can we make this use a font that has a 水 glyph, like the ic-unit-* tests, and have it on the root element? I feel like we're missing out on some test coverage by not having ric reference such a font on the root element.

And similarly for the rex test?

These tests feel like they're just checking that ric / rex compute against something other than the parent, but it would be good to check that they're computing against exactly what we want.

Reply:
Done!